server/aesdsocket_old.c

```diff
@@ -0,0 +1,201 @@
+#include <stdio.h>
```

Comment:
Delete old copies, use revision control to keep history rather than duplicate files.